dusty/cli/test.py

```diff
@@ -22,13 +23,13 @@
 
 from ..payload import Payload
 from ..commands.test import (run_app_or_lib_tests, test_info_for_app_or_lib, pull_repos_and_sync_commands,
-                             ensure_valid_suite_name)
+                             ensure_valid_suite_name, run_all_app_or_lib_suites)
 
 def main(argv):
     args = docopt(__doc__, argv, options_first=True)
     if not args['<suite_name>']:
         return Payload(test_info_for_app_or_lib, args['<app_or_lib_name>'])
-    else:
+    elif not args['--all']:
```

Comment:
Nitpick but generally it's easier to understand branches if you avoid negation whenever possible. So this could be `elif args['--all']` and switched with the else branch.